pallets/demeter-farming-platform/src/tests.rs

```diff
@@ -2394,17 +2394,6 @@ mod tests {
                     deposit_fee,
                 )
             );
-
-            let pool_infos =
```

Comment:
I think it should not be removed. It's presented in develop branch
https://github.com/sora-xor/sora2-network/blob/e469569e25150f244e6bb11d21a801b889e0d960/pallets/demeter-farming-platform/src/tests.rs#L2368